### DISCUSSION TITLE
Changeing network and security settings not posible in chromebook- linux installation

### DISCUSSION THREAD

**Discussion by sandro-gera** [rustdesk/rustdesk#6653] [General]
Hello everyone,
I installed rustdesk on my Chromebook in a Linux environment.
Unfortunately I have the problem that I cannot unlock the network and security settings.
Can anyone help me with this problem?

**Comment by bingooo** [comment]
You should give root access control permissions to open the display. then run rustdesk as root, modify the settings.

`xhost +si:localuser:root`
`sudo rustdesk`  now you can unlock the network and security settings.
`xhost -si:localuser:root`
